SteamReview/SteamReview/Category.h

```diff
@@ -12,14 +12,16 @@ class FCategory
 
 public:
 	FCategory();
-	FCategory(const std::string InCategoryName);
+	~FCategory();
+	FCategory(FCategory& OtherCategory);
+	FCategory(const std::string& InCategoryName);
 
-	bool AddGame(const FGame InGame);
-	FGame GetGame(const int GameIndex) const;
+	bool AddGame(const FGame& InGame);
+	bool GetGame(const int GameIndex, FGame& OutGame) const;
 
 	bool IsFull() const;
 	bool IsValid() const;
 
-	std::string GetCategoryName() const;
-	void DisplayCategory() const;
+	const std::string& GetCategoryName() const;
+	void DisplayCategory(FGame& OutGame) const;
```

Comment:
Now, this is really confusing, why would you have an out parameter in a method named DisplayCategory, if that method is named like that, I expect its behaviour will be restricted to displaying the category, not doing anything at all with the object and variables it deals with. I think that is also why this method is const, 'cause it's not supposed to modify anything, just display. It doesn't really make sense to add an out parameter to a const method, cause you won't even be able to modify it, it should be returned exactly how it was passed to the function.